scripts/isolateTests.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python
+#
+# This script reads C++ source files and writes all
+# multi-line strings into individual files.
+# This can be used to extract the Solidity test cases
+# into files for e.g. fuzz testing.
```

Comment:
Maybe include a comment that the intended use is to pull out the test cases from `test/libsolidity/*.cpp`.